src/Documentation/glossary.js

```diff
@@ -29,6 +29,13 @@ export default {
         'result (such as extracted features or a ML model file) that is ' +
         'under DVC control. Refer to [Data and Model Files Versioning]' +
         '(/doc/use-cases/data-and-model-files-versioning) for more details.'
+    },
+    {
+      name: 'Output',
+      match: ['output', 'outputs'],
+      desc:
+        'File or directory that is a result of `run` command.'
```

Comment:
this not exactly true. It can be import, dvc add, import-url also. It's good to mention these commands, but it's better to define it some user friendly way. What does output from a user perspective mean?